lib/src/value/list.ts

```diff
@@ -26,14 +32,18 @@ export class SassList extends Value {
    * `brackets`.
    */
   constructor(
```

Comment:
This should probably be defined using overloads, to reject `construct(ConstructorOptions, ConstructorOptions)` from valid calls in the type checker